src/components/PostRelatedArticles/index.jsx

```diff
@@ -0,0 +1,81 @@
+import { Link, graphql, useStaticQuery } from "gatsby"
+import PropTypes from "prop-types"
+import React from "react"
+
+import * as styles from "./index.module.scss"
+
+export const query = graphql`
+  {
+    allMarkdownRemark(
+      filter: { fileAbsolutePath: { regex: "/src.posts/" } }
+      sort: [{ frontmatter: { date: DESC } }]
+    ) {
+      nodes {
+        fields {
+          path
+        }
+        frontmatter {
+          categories {
+            key
+            label
+          }
+          title
+        }
+      }
+    }
+  }
+`
+
+function Article({ article }) {
```

Comment:
This is a bit nitpicking. But we are using `Post` through the project. Introducing `Article` might be a bit confusing.

I would also call this a `PostCard` probably